src/sentry/static/sentry/app/components/placeholder.tsx

```diff
@@ -16,13 +16,14 @@ type DefaultProps = Readonly<typeof defaultProps>;
 type Props = {
   className?: string;
   children?: React.ReactNode;
+  error?: React.ReactNode;
 } & Partial<DefaultProps>;
 
 const Placeholder = styled((props: Props) => {
-  const {className, children} = props;
+  const {className, children, error} = props;
   return (
     <div data-test-id="loading-placeholder" className={className}>
-      {children}
+      {error || children}
```

Comment:
as children and error are optional I would add something like 

```
if (!error && !children) {
return null
}
```

you never know if someone is not going to pass any  🤷‍♀️

Reply:
@priscilawebdev Placeholder should still render if both are not defined